contracts/evmscript/IEVMScriptRegistry.sol

```diff
@@ -9,9 +9,9 @@ import "./IEVMScriptExecutor.sol";
 
 contract EVMScriptRegistryConstants {
     /* Hardcoded constants to save gas
-    bytes32 public constant EVMSCRIPT_REGISTRY_APP_ID = apmNamehash("evmreg");
+    bytes32 internal constant EVMSCRIPT_REGISTRY_APP_ID = apmNamehash("evmreg");
     */
-    bytes32 public constant EVMSCRIPT_REGISTRY_APP_ID = 0xddbcfd564f642ab5627cf68b9b7d374fb4f8a36e941a75d89c87998cef03bd61;
+    bytes32 internal constant EVMSCRIPT_REGISTRY_APP_ID = 0xddbcfd564f642ab5627cf68b9b7d374fb4f8a36e941a75d89c87998cef03bd61;
```

Comment:
Was debating exposing this as part of `EVMScriptRegistry`; I didn't want to leave it `public` here because `AragonApp` inherits from this contract.

Reply:
Maybe we can make `getExecutorRegistry()` public in EVMScriptRunner?